eng/pipelines/libraries/outerloop-mono.yml

```diff
@@ -38,6 +38,9 @@ jobs:
         # extra steps, run tests
         extraStepsTemplate: /eng/pipelines/libraries/helix.yml
         extraStepsParameters:
+          scenarios:
+          - wasmtestonbrowser
+          - normal
```

Comment:
do we need both browser, and v8 run?

Reply:
I think we use v8 for most tests currently because it's lightweight. 
On the other hand, it doesn't have some features we use. This PR is mostly about networking and that's feature of browser, not feature of js engine. NodeJs has separate implementation AFAIK.

Reply:
Then can we use just browser here? IIUC, `wasmtestonbrowser` will run tests run with `normal`(v8)+the networking ones. Not objecting to it, just making sure that this is the intention.